lib/tools/adb-commands.js

```diff
@@ -727,6 +734,32 @@ methods.setWifiAndData = async function ({wifi, data}, isEmulator = false) {
 };
 
 /**
+ * Change the state of animation on the device under test.
```

Comment:
Can you please add a line that describes why this might be needed to change that state?